apps/web/src/pages/templates/components/LackIntegrationAlert.tsx

```diff
@@ -35,10 +35,24 @@ export function LackIntegrationAlert({
     useSelectPrimaryIntegrationModal();
 
   const onIntegrationModalClose = () => openIntegrationsModal(false);
+  const handleErrorRectangleClick = () => {
```

Comment:
@AnshuPathak-88825 you can remove the duplicate onClick code, if we are doing the whole div clickable 🙏